glotaran/builtin/io/pandas/test/test_pandas_parameters.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+
+from pathlib import Path
+
+from glotaran.io import load_parameters
+from glotaran.io import save_parameters
+
+PATH_XLSX = Path.cwd().joinpath("glotaran/builtin/io/pandas/test/data/parameter.xlsx")
+PATH_YAML = Path.cwd().joinpath("glotaran/builtin/io/pandas/test/data/parameter.yaml")
+DATA_PATH = Path("testparameters.xlsx")
+
+
+def test_load_parameters_xlsx():
+    """load parameters file as yaml and excel and compare them"""
+    parameters_xlsx = load_parameters(PATH_XLSX)
+    parameters_yaml = load_parameters(PATH_YAML)
+    assert parameters_yaml == parameters_xlsx
+
+
+def test_save_parameters_xlsx(
+    tmp_path: Path,
+):
+    """load parameters file from yaml and save as xlsx
+    and compare with yaml and reloaded xlsx parameter files"""
+    parameters_yaml = load_parameters(PATH_YAML)
+    parameter_path = tmp_path / DATA_PATH
```

Comment:
```suggestion
    parameter_path = tmp_path / "testparameters.xlsx"
```
See other suggestion in this file.